src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor/Functions/LongRunningTriggerFunction.cs

```diff
@@ -0,0 +1,49 @@
+using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Model;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.DurableTask;
+using Microsoft.DurableTask.Client;
+using Microsoft.Extensions.Logging;
+
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Functions;
+
+public class LongRunningTriggerFunction(
+    ILogger<LongRunningTriggerFunction> logger)
+{
+    [Function(nameof(TriggerLongRunningOrchestration))]
+    public async Task<IActionResult> TriggerLongRunningOrchestration(
```

Comment:
Given that it looks like we've proven that the status check works, is there any value anymore in having this code?

Would probably argue that we could just delete all this long-running function code as it's just cluttering up the project.

If we think there's a reason to keep it, would suggest that we at least [disable the trigger](https://learn.microsoft.com/en-us/azure/azure-functions/disable-function?tabs=portal) to prevent it from being triggered accidentally when we don't need it.

Reply:
Good idea, and I have disabled by adding into `deploy-data-processor.yml`:

```
"AzureWebJobs.TriggerLongRunningOrchestration.Disabled=true" \                  
```

I'd suggest we keep it for allowing the testing of this ticket, upon which it can be enabled in Azure Portal and triggered when needed.

I've also moved the `App__MetaInsertBatchSize` out of Bicep and into `deploy-data-processor.yml` as well, so now all app-specific settings are controlled in the yml.